src/libraries/System.Net.Http.Json/src/System.Net.Http.Json.csproj

```diff
@@ -21,7 +22,7 @@
     <Compile Include="System\Net\Http\Json\TranscodingReadStream.cs" />
     <Compile Include="System\Net\Http\Json\TranscodingWriteStream.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)' or '$(TargetFramework)' == 'net5.0'">
```

Comment:
nit: consider using a negation here to avoid hardcoding the previous netcoreapp version: `'$(TargetFramework)' != 'netstandard2.0' and '$(TargetFramework)' != 'net461'.